examples/binaries/mul-by-const/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 
 // This contract recursively composes itself with another contract (the other contract
 // being applied to the input data first): `c(f) = (c(f) . f) x`.
-// Every call to the auto_composer contract incremets the internal `ITER` counter.
+// Every call to the auto_composer contract increments the internal `ITER` counter.
```

Comment:
Looks like a description of some other contract (`demo-compose` most likely). Anyway, I will revisit it later and put a proper description

Reply:
Yeah, seems like this :S
I don't actually know how it appeared, but yeah, better to resolve in separate PR